filebeat/autodiscover/builder/hints/logs_test.go

```diff
@@ -44,6 +45,7 @@ func TestGenerateHints(t *testing.T) {
 					},
 				},
 			},
+			path:   "/var/lib/docker/containers/${data.kubernetes.container.id}/*-json.log",
```

Comment:
Could you keep the tests with ids and add some specific tests for the new option?